src/reviews/reviews.service.ts

```diff
@@ -30,15 +30,15 @@ export class ReviewsService {
   }
 
   async findOne(id: number): Promise<any> {
-    const review = await this.prisma.review.findUnique({where: {id}});
+    const review = await this.prisma.review.findUnique({ where: { id } });
```

Comment:
I guess error handling should be done here
If it throws error let's rename function to `findOneOrFail`